invenio_app_ils/acquisition/mappings/v6/acq_orders/order-v1.0.0.json

```diff
@@ -163,6 +163,34 @@
         },
         "vendor_pid": {
           "type": "keyword"
+        },
+        "vendor": {
+          "type": "object",
+          "properties": {
+            "pid": {
+              "type": "keyword"
+            },
+            "name": {
+              "type": "text",
+              "fields": {
```

Comment:
do we need the `name` field as keyword?

Reply:
Yes, we need it for the vendor facet on the order search page.